### ISSUE TITLE
Fake errors just after VSCode startup!

### ISSUE THREAD

**Issue by clintwood** [UnwrittenFun/svelte-vscode#101]
On opening a `.svelte` file in the editor after just starting `code-insiders` I consistently get an error on some other random `.svelte` file (not the one I have opened) like shown below.

![image](https://user-images.githubusercontent.com/2149388/73133700-1e0f8780-4035-11ea-87c9-d8f013754c40.png)

If I then open the offending `.svelte` file (as shown below) the errors all go away.

![image](https://user-images.githubusercontent.com/2149388/73133710-3bdcec80-4035-11ea-972b-d6abe09fffcb.png)

Once the offending file is visited these errors don't show up again until I close and open vscode again!

If I disable `vscode-svelte` these errors don't show (nor do I have any intellisense on the svelte files)!  I can't tell if this is a know problem or a config issue?

**Comment by hmmhmmhm** [comment]
In my experience, this plug-in does not seem to reinterpret the code that interpreted the structure once, so I recommend that you restart plugin it at regular intervals. (`Do not restart vscode, enter F1 and use the Restart Svelte Server command.`)

I hope that if this plug-in developer reads this article, i'll tell plz create an option to re-analyze the code just like the existing IDE.

**Comment by clintwood** [comment]
That's interesting - I just did that and that same initial error comes back - again if I open the offending file all the errors go away again - It would be nice to know what is going on between these extensions or rather which one is causing this error!

**Comment by hmmhmmhm** [comment]
I suspect that the `svelte.config.js` in the project may have been wrong.

Recently I understood that when this plug-in interprets svelte.config.js, an error occurs when I give preprocess option. so I set the preprocess option to apply only when it runs outside of vscode, and now I use the typescript intellisense well.

It might be helpful to check my `svelte.config.js` set-up.
https://github.com/hmmhmmhm/svelte-template/blob/master/svelte.config.js

**Comment by hmmhmmhm** [comment]
Oh wait, is this a situation where only some files get this error? I've been through it a few times lately. This is probably a plug-in error.

**Comment by clintwood** [comment]
@hmmhmmhm thanks for your responses!  I'm not using a preprocessor so I don't have a `svelte.config.js` - I added an empty config - same issue.

Yes it's randomly chooses a `.svelte` file and shows these errors but only when I visit (open) any other `.svelte` file for the first time but the errors are _not_ on the file I just opened for the first time they are on the randomly selected `.svelte` file.  If I visit this randomly erroring file the errors go away and the rest of my coding session is error free (unless I make errors of course)!

EDIT: It only ever errors once on one random `.svelte` file per editing session.
